billing/actions/accounts.py

```diff
@@ -329,3 +324,15 @@ def toggle_delinquent_status(account_ids: List[int]):
         When(delinquent=True, then=Value(False)),
         default=Value(True)
     ))
+
+
+def charge_pending_invoices(account_id: str):
+    account = Account.objects.get(id=account_id)
+    pending_invoices = account.invoices.payable().only('pk')
+    logger.info('charge-pending-invoices', pending_invoices=pending_invoices)
+    payments = [
```

Comment:
payment_transactions (instead of payments)

Reply:
Done